### PR TITLE
Localizes paths for newrelic.com

### PR DESCRIPTION
**Pull request by caylahamann**
Localizes external link paths to newrelic.com

### CODE REVIEW

File: packages/gatsby-theme-newrelic/src/utils/localization.js [hunk 1]
```diff
@@ -11,3 +11,16 @@ export const localizePath = ({ path, locale }) => {
...
 };
+
+export const localizeExternalLink = ({ link, locale }) => {
+  if (locale.isDefault) {
+    return link;
+  }
+
+  const paths = link.replace('https://', '').split('/');
+
+  paths.splice(1, 0, locale.locale).join('/');
```
Comment (jerelmiller) [inline packages/gatsby-theme-newrelic/src/utils/localization.js:22]: Rather than using array methods, which get just a little hard to follow, you could clean this up a little bit by using the `URL` constructor. Doing so also allows you to take advantage of the `localizePath` helper above.

```js
const url = new URL(link)

url.pathname = localizePath({ path: url.pathname, locale })

return url.href
```

**Review by jerelmiller — COMMENTED** [review]
Had one minor suggestion on constructing that pathname

File: packages/gatsby-theme-newrelic/src/utils/localization.js (hunk 1 above)
Comment (caylahamann) [reply packages/gatsby-theme-newrelic/src/utils/localization.js:22]: oooh yes, I'm good with that! Let me make that refactor

Comment (caylahamann) [reply packages/gatsby-theme-newrelic/src/utils/localization.js]: @jerelmiller It has been updated! tested it out with all the links and works great :)

**Review by jerelmiller — APPROVED** [review]
Pulled it down and saw the sign up button went to the localized sign up page. Huzzah!
